### PR TITLE
fix multi level module

### PR DESCRIPTION
**Pull request by windard**
Change-Id: I59002abfd3d2dd5665ddee8a6a12d1513f15758a

### CODE REVIEW

**Comment by aisk** [comment]
@windard do you have any related issue for this PR? And I think it should be cool if there was a test case for we don't break the behavior in this PR in the future.

File: thriftpy2/parser/parser.py [hunk 1]
```diff
@@ -403,7 +403,7 @@ def p_ref_type(p):
                 ref_type = resolved_ref_type
                 break
     else:
-        for index, name in enumerate(p[1].split('.')):
+        for index, name in enumerate(p[1].rsplit('.', 1)):
```
Comment (aisk) [inline thriftpy2/parser/parser.py:406]: The split count is set to one, so I thought this will not work with thrift files with multiple dot in name.

Comment (aisk) [reply thriftpy2/parser/parser.py:406]: And why change the split to rsplit? I think the resolve order is chaned and I thought it's a ad-hoc to your case and will not work with other kind of cases.

**Comment by windard** [comment]
yeah, it has been fixed, but not released. let's close this PR.
https://github.com/Thriftpy/thriftpy2/pull/125

**Pull request closed without merging**